web-client/integration-tests/journey/docketClerkCreatesARemoteTrialSession.js

```diff
@@ -69,13 +69,12 @@ export const docketClerkCreatesARemoteTrialSession = (test, overrides = {}) => {
 
     expect(test.getState('validationErrors')).toEqual({
       startDate: errorMessages.startDate[1],
-      term: errorMessages.term,
```

Comment:
Could you explain this? Shouldn't the intent be to create a validation error and prove that it is displayed?

Reply:
The thought was that since the `term` calculation is already being tested in `computeTrialSessionFormDataAction`, we could avoid passing a 13th month into moment, which returns an invalid object. In retrospect that seems backward.